src/types/utils/isNaN.d.ts

```
@@ -0,0 +1,3 @@
+declare function isNaN(value: any): boolean;
```

Biome

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)